draft-ietf-quic-recovery.md

```diff
@@ -1031,9 +1031,10 @@ OnAckReceived(ack, pn_space):
     OnPacketAcked(acked_packet.packet_number, pn_space)
 
   DetectLostPackets(pn_space)
-
+  // Immediately attempt to send, but never more than the
+  // initial window in a single burst.
+  SendUpToIWPacketsAsCongestionControllerAllows()
```

Comment:
What's the rationale for sending a burst here? And if the congestion controller allows more, why limit to IW?

Reply:
The intent is is not to send a burst, but rather to limit any sending to the IW, as specified in the normative text: 

"Implementations MUST either use pacing or limit such bursts to the initial congestion window, which is recommended to be the minimum of 10 * max_datagram_size and max(2* max_datagram_size, 14720)), where max_datagram_size is the current maximum size of a datagram for the connection, not including UDP or IP overhead."